src/views/Project/index.jsx

```diff
@@ -229,10 +372,12 @@ export default class Project extends Component {
           return previousResult;
         }
 
+        pagingGithub[searchQuery] = fetchMoreResult.search.pageInfo;
+
         return dotProp.set(
           previousResult,
           'search.nodes',
-          moreNodes.concat(previousResult.search.nodes)
+          previousResult.search.nodes.concat(moreNodes)
```

Comment:
Any reason for this change? Since this PR is quite large in size, let's strive to minimize the number of lines changed as much as we can.